scripts/restore

```diff
@@ -50,19 +47,21 @@ ynh_restore
 #=================================================
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
-yunohost service add $app
 systemctl enable $app.timer --quiet
 systemctl start $app.timer
-#yunohost service add $app.timer
-#yunohost service enable $app.timer
-#yunohost service start $app.timer
```

Comment:
If those ain't needed anymore, let's remove them entirely instead of keeping them as comments...